src/Microsoft.AspNetCore.Authentication.OpenIdConnect/Events/OpenIdConnectEvents.cs

```diff
@@ -27,10 +27,12 @@ public class OpenIdConnectEvents : RemoteAuthenticationEvents
         public Func<MessageReceivedContext, Task> OnMessageReceived { get; set; } = context => Task.CompletedTask;
 
         /// <summary>
-        /// Invoked before redirecting to the identity provider to authenticate. This can be used to set ProtocolMessage.State
-        /// that will be persisted through the authentication process. The ProtocolMessage can also be used to add or customize
+        /// Invoked before redirecting to the identity provider to authenticate. This can be used to set
+        /// <see cref="RedirectContext.ProtocolMessage"/>
+        /// that will be persisted through the authentication process. The ProtocolMessage can also be used to add or alter
```

Comment:
What were you trying to convey? Or did you just want to add the cref? 

This is confusing two ideas. This sentence was specifically about the ability to round trip user State. Customizing the RedirectContext.ProtocolMessage is already covered in the next sentence.